tox.ini

```
@@ -25,6 +25,13 @@ commands =
     pyflakes ./docs/conf.py
 deps = flake8
 
+[testenv:format]
+skipsdist = True
+commands =
+    black --version
+    black --check .
```

`--diff` will also print out the difference, which can save some time if the PR author doesn't have Black installed:

```suggestion
    black --check --diff .
```